composer.json

```diff
@@ -22,6 +22,6 @@
   "minimum-stability": "beta",
   "prefer-stable": true,
   "scripts": {
-    "install-blt-alias": "./blt.sh install-alias"
+    "install-blt-alias": "blt install-alias"
```

Comment:
This is actually one place we need to refer to the script explicitly, since it would only be run if the alias weren't already installed.